setup.py

```diff
@@ -101,4 +101,5 @@
             'yapf = pyls.plugins.yapf_format'
         ]
     },
+    python_requires='>=3,<=3.7.0'
```

Comment:
This may require some further discussion, as I am not sure if allowing a pip install of a non compatible Python version is very smart